### PR TITLE
Initialized parent login and sign up

### PR DESCRIPTION
**Pull request by jimzhaozhengwang**
Dependency on https://github.com/jimzhaozhengwang/CS446-Project/pull/5

### CODE REVIEW

File: android/app/src/main/java/com/chorestory/app/ParentLoginSignUpActivity.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        loginButton.setEnabled(false);
+        signUpButton.setEnabled(false);
+    }
+
+    private void enableButtons() {
+        loginButton.setEnabled(true);
+        signUpButton.setEnabled(true);
+    }
+
+    private void navigateToParentLogin() {
```
Comment (keller00) [inline android/app/src/main/java/com/chorestory/app/ParentLoginSignUpActivity.java]: These 2 functions could be replaced by:
```
private void navigateTo(Class<?> cls){
    intent = new Intent(this, cls);
    startActivity(intent);
}
```

Or even by:
```
private void navigateTo(Class<?> cls){
    startActivity(new Intent(this, cls));
}
```

Then could be called like this:
```
navigateTo(ParentLoginActivity.class);
```

What do you think about that?

File: android/app/src/main/java/com/chorestory/app/ParentLoginSignUpActivity.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    @Override
+    protected void onResume() {
+        super.onResume();
+        enableButtons();
+    }
+
+    // prevent users from spam clicking buttons
+    private void disableButtons() {
+        loginButton.setEnabled(false);
```
Comment (keller00) [inline android/app/src/main/java/com/chorestory/app/ParentLoginSignUpActivity.java]: Could we maybe make a parent class that implements `disableButtons ` and `enableButtons` with an abstract linked list of the buttons that each class would have to overwrite? That would make a lot of this boilerplate code go away.

If we did that we could also integrate the `onResume` and `navigateTo` into it, they seem to be copy pastas in each of these files.

**Comment by isabelleleeson** [comment]
Can we include screenshots when doing UI changes?

**Comment by keller00** [comment]
Screenshots:
Screenshot_1561182280
Screenshot_1561182286
Screenshot_1561182289
Screenshot_1561182293

**Pull request merged**